src/amber/router/router.cr

```diff
@@ -32,7 +32,7 @@ module Amber
         trail = build_node(route.verb, route.resource)
         node = @routes.add(route.trail, route)
         @routes_hash["#{route.controller.downcase}##{route.action.to_s.downcase}"] = route
-        add_head(route) if route.verb == :GET
+        add_head(route) if route.verb == "GET"
```

Comment:
Was this not working as a symbol? I can't imagine that it was, unless there is a mapping somewhere.

Reply:
Correct, didn't work as a symbol.

Reply:
Definitely was not working since `route.verb` is a string type